lib/elements/number.js

```diff
@@ -55,7 +57,21 @@ class NumberPrompt extends Prompt {
       this.placeholder = false;
       this.rendered = this.transform.render(`${round(v, this.round)}`);
       this._value = round(v, this.round);
+      
+      if (this.resetDelay > 0) {
+        clearTimeout(this.resetTimer);
+        this.resetTimer = setTimeout(() => {
+          const color = this.color;
+          this.color = `inverse`;
+          this.render();
+          setTimeout(() => {
+            this.color = color;
+            this.reset();
+          }, 200);
+        }, this.resetDelay);
+      }
```

Comment:
You're correct, it does change the behavior to **always reset**. It also turns the reset timer **off**, by default. I'm open to changing it back to the original-ish behavior if it would improve this PR's chance of being accepted, but I suppose I'm still a little confused about the purpose of this feature.

I changed the behavior with the assumption that if the input was to be automatically erased, the user would want to understand their time limit. The original behavior "felt" random and inconsistent to me given that the act of typing a number could involve activity that would take more time than the delay.

On the other hand, the original behavior had the benefit of preserving the last-typed number indefinitely if no new input was recorded.